spline-shadow/rest-gateway/src/main/scala/za/co/absa/spline/gateway/rest/controller/AboutController.scala

```diff
@@ -16,23 +16,21 @@
 
 package za.co.absa.spline.gateway.rest.controller
 
+import com.typesafe.config.{ConfigFactory, ConfigRenderOptions}
 import io.swagger.annotations._
+import org.springframework.http.MediaType._
 import org.springframework.web.bind.annotation.{GetMapping, RequestMapping, RestController}
-import za.co.absa.spline.common.ARM.managed
-
-import scala.io.Source.fromInputStream
+import za.co.absa.spline.common.SplineBuildInfo
 
 @RestController
 @RequestMapping(Array("/about"))
 class AboutController {
 
-  @GetMapping(path = Array("/build"), produces = Array("text/x-java-properties"))
+  @GetMapping(path = Array("/build"), produces = Array(APPLICATION_JSON_VALUE))
   @ApiOperation("Get application version and build information")
```

Comment:
You could name the endpoint appVersionAndBuildInfo and then the comment would be redundant

Reply:
This is not a comment, it's part of Swagger documentation. It's supposed to be verbose.
API itself is already concise and self explanatory - /about/build